src/core/trulens/core/database/sqlalchemy.py

```diff
@@ -554,9 +554,21 @@ def insert_feedback(
             ):
                 session.merge(_feedback_result)  # update existing
             else:
-                session.merge(
-                    _feedback_result
-                )  # insert new result # .add was not thread safe
+                if (
+                    self.engine.dialect.name == "snowflake"
+                    and _feedback_result.result is None
+                ):
+                    # The Snowflake stored procedure connector isn't currently
+                    # capable of handling None qmark-bound to an `INSERT INTO`
+                    # statement (even for a nullable column). Thus, as a hack,
+                    # we get around this by first inserting a non-null value
+                    # then updating it to a null value.
+                    _feedback_result.result = -1
+                    session.merge(_feedback_result)  # .add was not thread safe
+                    _feedback_result.result = None
+                    session.merge(_feedback_result)
```

Comment:
This is because update into supports null ?

Reply:
No apparently not, so I've changed it and added a test now for it.

Earlier when I thought it worked it turned out I only did a `snowflake_session.sql("UPDATE...")` and not a `snowflake_session.sql("UPDATE...").collect()` so I thought it worked haha.